docs/dev/table/connect.md

```diff
@@ -58,11 +56,7 @@ The following table list all available connectors and formats. Their mutual comp
 | JSON              | `flink-json`                 | [Download](http://central.maven.org/maven2/org/apache/flink/flink-json/{{site.version}}/flink-json-{{site.version}}-sql-jar.jar) |
 | Apache Avro       | `flink-avro`                 | [Download](http://central.maven.org/maven2/org/apache/flink/flink-avro/{{site.version}}/flink-avro-{{site.version}}-sql-jar.jar) |
 
-{% else %}
-
-This table is only available for stable releases.
-
-{% endif %}
```

Comment:
Why did you remove the if/else?

Reply:
You are using docs of an unstable release. Why should this be confusing? Please undo these changes.

Reply:
@twalthr I have recovered the changes, can you help me to have a review, thank you!